redis.conf

```diff
@@ -1929,6 +1929,7 @@ hash-max-listpack-value 64
 # -3: max size: 16 Kb  <-- probably not recommended
 # -2: max size: 8 Kb   <-- good
 # -1: max size: 4 Kb   <-- good
+# Note that less than -5 has the same effect as -5.
```

Comment:
i'm not certain we wanna make this official (by documenting it),
people should just use documented values.

Reply:
Removed